### PR TITLE
Disable coverage build by default + add respective cmdline param

### PR DESCRIPTION
**Pull request by BjoernAtBosch**

### CODE REVIEW

File: build-system/cpp-cmake-conan/src/build.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def build(
     host_arch: str,
     build_target: str,
     static_build: bool,
-    coverage: bool = True,
+    coverage: bool = False,
```
Comment (MP91) [inline build-system/cpp-cmake-conan/src/build.py:91]: default value is not really needed anymore since the argparser already stores default values

**Review by MP91 — APPROVED** [review]
lgtm

File: build-system/cpp-cmake-conan/src/build.py (hunk 1 above)
Comment (BjoernAtBosch) [reply build-system/cpp-cmake-conan/src/build.py:91]: Who knows where it's needed ;-)

Comment (MP91) [reply build-system/cpp-cmake-conan/src/build.py:91]: yep thats why I still approved it :)

**Pull request merged**